### PR TITLE
🇳🇱🇩🇰🇸🇪🇨🇿🇮🇹 Add deployments for all the countries in prod and integration

### PR DESCRIPTION
**Pull request by CharlesIC**

### CODE REVIEW

File: ci/prod/deploy-pipeline.yaml [hunk 1]
```diff
@@ -147,3 +147,291 @@ spec:
...
+          platform: linux
+          image_resource: *task_toolbox
+          inputs:
+          - name: manifests
+          params:
+            KUBERNETES_SERVICE_ACCOUNT: ((namespace-deployer))
+            KUBERNETES_TOKEN: ((namespace-deployer.token))
+            KUBERNETES_API: kubernetes.default.svc
+            RELEASE_NAMESPACE: ((namespace-deployer.namespace))
+            RELEASE_NAME: nl
```
Comment (Wynndow) [inline ci/prod/deploy-pipeline.yaml]: Should this be `se`?

File: ci/prod/deploy-pipeline.yaml [hunk 2]
```diff
@@ -147,3 +147,291 @@ spec:
...
```
Comment (Wynndow) [inline ci/prod/deploy-pipeline.yaml]: `cz`?

File: ci/prod/deploy-pipeline.yaml [hunk 3]
```diff
@@ -147,3 +147,291 @@ spec:
...
```
Comment (Wynndow) [inline ci/prod/deploy-pipeline.yaml]:
```suggestion
            RELEASE_NAME: it
```

**Review by Wynndow — APPROVED** [review]

**Review by tommrr — APPROVED** [review]
LGTM

**Review by idrop — APPROVED** [review]

**Pull request merged**